### PR TITLE
fix option --keep_exact_zeros

### LINKED ISSUE #246: running with --keep_exact_zeros crashes
Running omc3 with the option --keep_exact_zeros crashes with a `ValueError: Index data must be 1-dimensional` 
issue stems from  https://github.com/pylhc/omc3/blob/f530c921deaab5f35a071620ce83bf5e76fef5a7/omc3/harpy/clean.py#L124 as it return a `pd.DataFrame` and not the required empty `pd.Index` 
exact zeros check is also skipped in the accuracy tests

### PR DESCRIPTION
**Pull request by mihofer**
Fixes #246

### CODE REVIEW

**Comment by mihofer** [comment]
Currently, the extended test options only include different windows and keep_zeros yes/no .. can be extended (by someone else) if needed/desired

File: omc3/harpy/clean.py [hunk 1]
```diff
@@ -140,14 +140,14 @@ def _get_bad_bpms_summary(harpy_input, known_bad_bpms, bpm_flatness, bpm_spikes,
...
+        raise ValueError("Total Number of BPMs after filtering is zero.")
     n_bad_bpms = n_total_bpms - n_good_bpms
     LOGGER.debug(f"(Statistics for file reading) Total BPMs: {n_total_bpms}, "
                  f"Good BPMs: {n_good_bpms} ({(100 * n_good_bpms / n_total_bpms):2.2f}%), "
                  f"Bad BPMs: {n_bad_bpms} ({(100 * n_bad_bpms / n_total_bpms):2.2f}%)")
     if (n_good_bpms / n_total_bpms) < 0.5:
         raise ValueError("More than half of BPMs are bad. "
-                         "This could be cause a bunch not present in the machine has been "
+                         "This could be because a bunch not present in the machine has been "
                          "selected or because a problem with the phasing of the BPMs.")
```
Comment (lmalina) [inline omc3/harpy/clean.py:151]: then also "because of a problem"

File: omc3/harpy/clean.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def clean(harpy_input, bpm_data, model):
     """
     bpm_data, bpms_not_in_model = _get_only_model_bpms(bpm_data, model)
     if bpm_data.empty:
-        raise AssertionError("Check BPMs names! None of the BPMs was found in the model!")
+        raise AssertionError("Check BPMs names! None of the BPMs were found in the model!")
```
Comment (lmalina) [inline omc3/harpy/clean.py:35]: This is wrong, change it back.

**Review by lmalina — CHANGES_REQUESTED** [review]

File: omc3/harpy/clean.py (hunk 2 above)
Comment (mihofer) [reply omc3/harpy/clean.py:35]: https://data.grammarbook.com/blog/singular-vs-plural/none-were-vs-none-was/

Comment (lmalina) [reply omc3/harpy/clean.py:35]: This exists only in US english.

**Review by lmalina — APPROVED** [review]
Putting English aside.

**Pull request merged**